src/main/java/uk/gov/hmcts/reform/cpo/data/CasePaymentOrderEntity.java

```diff
@@ -49,4 +49,7 @@
     @Column(length = 70)
     private String createdBy;
 
+    @Column(length = 70)
+    private Boolean historyExists;
```

Comment:
probably primitive boolean as we always have a value?

Reply:
Will change now, wasnt sure if the entity would accept primitives as i know some sometimes only accept the object